src/Storage.re

```diff
@@ -7,11 +7,16 @@ type config = {. [@bs.get nullable] "projectId": string};
 let make = (~config=?, ()) => make(config);
 
 [@bs.send] external bucket : (t, string) => Bucket.t = "";
-/* let s = make(); */
-/* let b = bucket(s, "m.lidco.io"); */
-/* let f = */
-/*   Bucket.file( */
-/*     b, */
-/*     "4eef6f7783b8b708080c8f2f7d7087ed62eda9b1/9a07a146-6fcb-3531-bfd2-d46bbe4cecbd.wav", */
-/*   ); */
-/* File.exists(f, (_, exists) => Js.log(exists)); */
+
+let s = make();
+
+let b = bucket(s, "m.lidco.io");
+
+let f =
+  Bucket.file(
+    b,
+    "4eef6f7783b8b708080c8f2f7d7087ed62eda9b1/9a07a146-6fcb-3531-bfd2-d46bbe4cecbd.wav",
+  );
+
+/* doesn't compile */
+File.exists(f, (_, exists) => Js.log(exists));
```

Comment:
callbacks are annotated with the guaranteed uncurrying thing so I believe in Reason syntax it would be something like:
```
File.exists(f, (. _, exists) => Js.log(exists));
```
If you want to go the `Callback` route, tho, I would do the following:
```
open Callback

let computation =
    File.exists(f) >> (exists) =>
      Js.log(exists);
> computation : Callback.t(unit)

let () =
  Callback.finish(computation);
```
This will take care of setting the pipes the right way and throwing an error if it is returned.